modules/nf-core/cellranger/vdj/main.nf

```diff
@@ -0,0 +1,54 @@
+process CELLRANGER_VDJ {
+    tag "$meta.gem"
```

Comment:
Why are you introducing meta.gem here? The standard syntax is id.

Reply:
that's how the `cellranger count` module is coded:

https://github.com/nf-core/modules/blob/ffaff8089fa2dec69e52a18c72384eee32bea2c4/modules/nf-core/cellranger/count/main.nf#L2

happy to change it to `"${meta.id}"` but in that case we should consider updating `cellranger count` too

Reply:
I'm inclined to still update `cellranger count` with `${meta.gem} --> ${meta.id}` since the latter is still available for that nf-core/scrnaseq workflow

maybe one of @grst @apeltzer can give a secondary opinion? 🙏🏼

Reply:
`gem` is used if a sample was sequenced across multiple flow cells. In that case cellranger expects multiple input folders. Each folder has a separate `id`, but the same `gem`. 

I think this could/should be solved differently by renaming input files, see also issues
https://github.com/nf-core/scrnaseq/issues/147
https://github.com/nf-core/scrnaseq/issues/195

Reply:
ok @grst fair to say that we should leave `${meta.gem}` alone for now?

we can always update the modules later once consensus is reached about input file names